### DISCUSSION TITLE
Zigbee Wind speed sensor

### DISCUSSION THREAD

**Discussion by lboue** [espressif/arduino-esp32#10459] [Question - Community Help]
Hi,

I am trying to implement a Zigbee Wind speed sensor example in PR#10455.
I am looking for Zigbee DeviceID for wind speed sensors.

Do I have to use `HA Simple sensor`?
![image](https://github.com/user-attachments/assets/1315aa53-9467-4f59-b493-09f0fdef2efd)

Regards

**Comment by P-R-O-C-H-Y** [comment]
Hi @lboue, from what I know and searched, there is no ZHA ID for wind speed sensor. Use the HA Simple sensor as you pointed out :)
